code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -364,6 +538,30 @@
 		"Vox" = 'icons/mob/species/vox/suit.dmi'
 		)
 
+//trackjackets
+
+/obj/item/clothing/suit/tracksuit
+	name = "black tracksuit"
+	desc = "Lightweight and stylish. What else could a man ask of his tracksuit?"
+	icon_state = "trackjacket_open"
+	item_state = "bltrenchcoat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	ignore_suitadjust = 0
+	suit_adjusted = 1
+	action_button_name = "Open/Close Jacket"
+	adjust_flavour = "unzip"

```

Comment:
Can go ahead and add the Vox species fitting stuff here too once the new sprites are added